### PR TITLE
Added Auth For Login and sign up

### PR DESCRIPTION
**Pull request by jeetbhalani15**
## Added Auth With  Login & Signup Functionality
-Added Login Widget Using API.
      =>login with guest credits.
-Added Signup Widget with API call
      =>Sign up with form validation.

 PR may look long due to package json files

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]

**Comment by Jay-Gadhiya** [comment]
- UI looks good
- code written well
- folder structure is good

File: Gravity-Notes/src/App.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import "../src/Default.css";
 import { Bodycontent } from "./Components/Body-content/Body-content";
 import { Footer } from "./Components/Footer/Footer";
 import { Navbar } from "./Components/Navigation/Navbar";
+import { Login } from "./Pages/AuthWidget/Login/Login";
+import { SignUp } from "./Pages/AuthWidget/SignUp/SignUp";
 import { Homepage } from "./Pages/Homepage/Homepage";
 import { Landingpage } from "./Pages/Landing-page/Landing-page";
```
Comment (Koushith) [inline Gravity-Notes/src/App.js:11]: you can create an index file and export all components/pages there. here you can import in single line.

File: Gravity-Notes/src/Pages/AuthWidget/SignUp/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    return errFlag;
+  };
+
+  const signupHandler = async (e) => {
+    e.preventDefault();
+    setError("");
+    if (formValidation()) {
+      console.log("hi");
+      try {
+        console.log("try");
```
Comment (Koushith) [inline Gravity-Notes/src/Pages/AuthWidget/SignUp/SignUp.js:142]: you can remove logs while pushing to prod.

**Review by Koushith — APPROVED** [review]
I think you have pushed node modules, you can add that in .gitignore file.

**Pull request merged**